packages/components/psammead-media-player/src/Amp/__snapshots__/index.test.jsx.snap

```diff
@@ -20,9 +20,11 @@ exports[`Media Player: Amp should render an amp-iframe with an amp-img nested in
         title="Media player"
       >
         <amp-img
+          attribution=""
```

Comment:
Where has this attribute come from?

Reply:
The attribution came about due to using this package https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/README.md#ampimg-1 instead of `amp-img`